packages/desktop/views/dashboard/account-management/AccountManagement.svelte

```diff
@@ -9,12 +53,59 @@
     >
         <div class="flex space-x-4 max-w-7xl justify-center w-full">
             {#key $selectedWallet?.id}
-                <left-pane class="flex flex-col w-1/3">
-                    <Pane height={Height.Full}>Account List</Pane>
-                </left-pane>
-                <right-pane class="w-full h-full min-h-96 flex-1 space-y-4 flex flex-col">
-                    <Pane height={Height.Full}>Account Details</Pane>
-                </right-pane>
+                {#if features.accountManagement.accountList.enabled}
```

Comment:
I would put the `if` of the `feature` inside the pane so that the view does not go out of alignment